udata_front/theme/gouvfr/assets/js/types.ts

```diff
@@ -114,43 +113,9 @@ export type Harvest = {
   backend: string;
 }
 
-export type NewDataset = Owned & {
-  title: string;
-  acronym: string;
-  archived: boolean;
-  description: string;
-  tags: Array<string> | null;
-  license: string;
-  frequency: string;
-  temporal_coverage: string;
-  frequency_date: Date | null;
-  page: string;
-  private: boolean;
-  quality?: Quality;
-  spatial: {
-    zones?: Array<string>;
-    granularity?: string;
-  } | null;
-};
+export type OwnedWithId = { organization: string, owner: never | null } | { organization: never | null, owner: string };
 
-export type Dataset = NewDataset & {
```

Comment:
Multiple components must be updated now that `Dataset` isn't exported from this file

Reply:
- CardMD
- CardMDList
- UserDatasetList
- Step4CompleteThePublication